internal/config/specs.go

```diff
@@ -13,13 +13,18 @@ type EnvSpec struct {
 
 	Debug bool `envconfig:"debug" default:"false"`
 
-	KratosPublicURL string `envconfig:"kratos_public_url" required:"true"`
-	KratosAdminURL  string `envconfig:"kratos_admin_url" required:"true"`
-	HydraAdminURL   string `envconfig:"hydra_admin_url" required:"true"`
+	KratosPublicURL     string `envconfig:"kratos_public_url" required:"true"`
+	KratosAdminURL      string `envconfig:"kratos_admin_url" required:"true"`
+	HydraAdminURL       string `envconfig:"hydra_admin_url" required:"true"`
+	OathkeeperPublicURL string `envconfig:"oathkeeper_public_url" required:"true"`
 
 	IDPConfigMapName      string `envconfig:"idp_configmap_name" required:"true"`
 	IDPConfigMapNamespace string `envconfig:"idp_configmap_namespace" required:"true"`
 
 	SchemasConfigMapName      string `envconfig:"schemas_configmap_name" required:"true"`
 	SchemasConfigMapNamespace string `envconfig:"schemas_configmap_namespace" required:"true"`
+
+	RulesConfigMapName      string `envconfig:"rules_configmap_name" required:"true"`
+	RulesConfigMapNamespace string `envconfig:"rules_configmap_namespace" required:"true"`
+	RulesFilesLocations     string `envconfig:"rules_file_locations" required:"true"`
```

Comment:
```suggestion
	RulesFilesLocations     []string `envconfig:"rules_file_locations" required:"true"`
```

Reply:
how do you pass a list as an environment variable?

Reply:
same way as you are doing manually now, splitting by commas https://github.com/kelseyhightower/envconfig/blob/v1.4.0/envconfig.go

check the docs for better examples

Reply:
The env var will not be used. Instead I'm using a function that is a one way conversion from an ID to a filename. All rules will have their own files. The files are prefixed by ADMIN_RULE_. Only rules originating from admin-ui can be edited.